src/Extension/Task/SnifferTaskSet.yaml

```diff
@@ -1,7 +1,7 @@
 ---
 id: "validation:php:all"
 short_description: "Validates your php code using different approaches like codestyle, architecural and static analysis"
-help: "Validates your php code using different approaches like codestyle, architecural and static analysis"
+help: "Validates your php code using different approaches like codestyle, architecural and static analysis. You may need to set default phpcs standard before using it. Use the next as an example 'vendor/bin/phpcs --config-set default_standard SPR12'"
```

Comment:
Looks like it requires acceptance tests improvements because they working not in the project root and don't understand      <rule ref="vendor/spryker/code-sniffer/Spryker/ruleset.xml"/>

Reply:
@stereomon can we specify phpcs.xml as the default --config option instead of removing it?

Reply:
If that helps solve the issue faster, yes. But @dereuromark said we shouldn't have that.

Reply:
I am not quite sure why you cannot use normal (default) root setup here.
But if it doesnt work any other way, you can use your config setting of course.
It might not always work the way it is now designed to work however, using project root setup should be the preferred way.

Reply:
@dereuromark I don't like that test dictates what I should do. I believe acceptance tests should be changed, but I don't know how to do it. They(tests) read default config file but they not in the project root directory and path 'vendor/....' isn't valid for them

Reply:
"I don't like that test dictates what I should do"
They shouldnt.
I just dont understand the problem here it seems.

The tests seem for me to be set up wrongly then. They should get their info from the project in a way that paths are clear.
If thats not possible things might have to get mocked.

Reply:
acceptance tests. impossible to mock.

Reply:
Another question:
Why is this using Acceptance in the first place?
I find them way less suited for CLI command testing than normal functional ones

Directly testing the command and its output based on the CLI arguments going in is not an option here?

Reply:
Test simple executes a command and checks output, report, etc. But command works incorrectly due to path changes. for the test `vendor` doesn't exist, because test under `tests/path/to` directory, so the rule path should be `../../../vendor/...` and I don't see any option how to replace it with another config file.

Reply:
Before that, I was able to change it by running tasks with a specific option. Currently, it is not possible

Reply:
Anyway, I skipped them. Such a pain in the socks from nothing...

Reply:
But again, why Acceptance and not normal Functional tests using codecept or phpunit?
Those suffice IMO and thats how I always test my commands.

Reply:
codeception

Reply:
We do not test command directly, we test task which executes a command in a subprocess